cluster-autoscaler/processors/status/scale_up_status_processor.go

```diff
@@ -31,6 +34,8 @@ type ScaleUpStatus struct {
 	PodsTriggeredScaleUp    []*apiv1.Pod
 	PodsRemainUnschedulable []NoScaleUpInfo
 	PodsAwaitEvaluation     []*apiv1.Pod
+	CreateNodeGroupResults  []nodegroups.CreateNodeGroupResult
+	ConsideredNodeGroups    map[string]cloudprovider.NodeGroup
```

Comment:
nit: why not []cloudprovider.NodeGroup? Seems like it would be simpler

Reply:
No preference here, changed